packages/pv-create-component/.eslintrc

```diff
@@ -0,0 +1,9 @@
+{
```

Comment:
why does this package need a own eslintrc?

Reply:
there are some additional rules which were needed here (mostly regarding template literals) and it made more sense to define them for this package only instead of all packages.